packages/pigment-css-core/src/css.js

```diff
@@ -0,0 +1,8 @@
+import { generateErrorMessage } from './utils';
+
+export default function css() {
+  console.error(generateErrorMessage('css'));
```

Comment:
Wouldn't it make more sense to throw here? Users may miss the error message and be puzzled about why no styles appear even though it seemingly is working. Besides that, wouldn't this produce a ton of duplicate messages in the console?